<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1546</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TCGA-Tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>102</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Choose Data:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="project_choice_menu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="calculate_correlation_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Correlation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gene Targets List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_targets_list_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>From file...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_targets_list_button">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save...</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="gene_targets_editable_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>211</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QCheckBox" name="z_cutoff_checkbox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>z-score cut-off</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="log2_checkbox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>log2-transform</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_heatmap_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>310</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Heatmap</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="z_transform_checkbox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>z-transform</string>
    </property>
   </widget>
   <widget class="QPushButton" name="find_directory_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>132</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Directory..</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extract_targets_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract Genes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_project_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>132</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QListWidget" name="metadata_display_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="loaded_targets_list">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>171</width>
      <height>301</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="plot_hist_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot Histogram</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_cutoff">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QWidget" name="mplwindow" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>721</width>
      <height>551</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="mplvl">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
    </layout>
   </widget>
   <widget class="QPushButton" name="generate_scatterplot_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Scatter Plot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transform_data_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Transform!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lmplot_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>lmPlot</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="log_display_box">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="correlation_output_box">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>341</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
    <property name="centerOnScroll">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="include_controls_checkbox">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Include controls?</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>project_choice_menu</zorder>
   <zorder>calculate_correlation_button</zorder>
   <zorder>label_2</zorder>
   <zorder>load_targets_list_button</zorder>
   <zorder>save_targets_list_button</zorder>
   <zorder>gene_targets_editable_list</zorder>
   <zorder>z_cutoff_checkbox</zorder>
   <zorder>log2_checkbox</zorder>
   <zorder>generate_heatmap_button</zorder>
   <zorder>z_transform_checkbox</zorder>
   <zorder>find_directory_button</zorder>
   <zorder>extract_targets_button</zorder>
   <zorder>load_project_button</zorder>
   <zorder>metadata_display_box</zorder>
   <zorder>loaded_targets_list</zorder>
   <zorder>plot_hist_button</zorder>
   <zorder>z_cutoff</zorder>
   <zorder>mplwindow</zorder>
   <zorder>generate_scatterplot_button</zorder>
   <zorder>transform_data_button</zorder>
   <zorder>lmplot_button</zorder>
   <zorder>log_display_box</zorder>
   <zorder>correlation_output_box</zorder>
   <zorder>include_controls_checkbox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1546</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
